pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -315,6 +315,11 @@ in
     # The ruby build script takes care of this
     dontUseCmakeConfigure = true;
 
+    postInstall = ''
```

Comment:
It might be useful for future archeologists to have the commit message also as a comment here.